include/actions.js

```diff
@@ -883,15 +883,15 @@ CodeBootVM.prototype.execute2 = function (single_step) {
         }
         catch (e) {
             console.log(e);//TODO: remove
-            update_playground_visibility();
+            update_playground_visibility();//TODO: fix
```

Comment:
Fix what?

Reply:
a reminder for myself... it needs to be passed the `vm`